tests/shared/models/cb-admin.test.js

```diff
@@ -0,0 +1,142 @@
+const test = require('tape');
+const pg = require('pg');
+const { getConfig } = require('../../../config');
+const { refresh: refreshDB } = require('../../../db/scripts');
+const cbAdmin = require('../../../react-backend/shared/models/cb-admin');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('Db Query | cbAdmin ', async tape => {
+  const client = new pg.Client(config.psql);
+  await client.connect();
+
+  tape.test('cbAdmin.getFeedback | existing cb_id', async t => {
```

Comment:
Since they're nested inside the `CbAdmin model` test suite, there's no need to prefix the method names with `CbAdmin`, since that is implicit.

Reply:
🤷🏽‍♀️ Dunno why but I find this easier to read/reason about. Would prefer to leave it as is if that's okay.